meilisearch/models/client.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
```

Comment:
I know this file is linked to the `client.py` file but wouldn't it be better to call it `key.py`? It's in the case of adding other classes that you decided to call it client instead? WDYT?

Reply:
Sure, I can do that